assets/pulpcore-api.prep

```diff
@@ -6,6 +6,9 @@ if -n {
 foreground {
   mkdir -p /etc/pulp/certs
 }
```

Comment:
Users may want to provide the db encryption key but not the token key.

So please use a separate checks for each of these, and create if they do not exist:
1. /etc/pulp/certs dir
2. the database key
3. the token key.